### DISCUSSION TITLE
`llama.cpp` is everywhere

### DISCUSSION THREAD

**Discussion by foldl** [ggml-org/llama.cpp#10746] [Show and tell]
I believe [Wolfram Research](https://www.wolfram.com/llm-benchmarking-project) is using `llama.cpp` for benchmarking when seeing those `Q4_K_M` suffices.

![image](https://github.com/user-attachments/assets/733e09a5-a3ea-400a-9040-d9392ae9e9ab)

**Comment by ExtReMLapin** [comment]
We should take a moment to appreciate llama.cpp is under MIT license.
